go-controller/pkg/util/multi_network.go

```diff
@@ -337,6 +344,14 @@ func (layer2NetConfInfo *Layer2NetConfInfo) MTU() int {
 	return layer2NetConfInfo.mtu
 }
 
+func (layer2NetConfInfo *Layer2NetConfInfo) Subnets() []string {
+	subnets := strings.Split(layer2NetConfInfo.subnets, ",")
+	if len(subnets) == 1 && subnets[0] == "" {
```

Comment:
I cannot prevent users from shooting themselves in the foot. That's a wrong configuration.

Reply:
thinking this again, it costs virtually nothing to trim the whitespaces, and provide the user with a decent error msg, instead of leaving them scratching their heads.